site/docs/guide/api.md

```diff
@@ -132,3 +183,99 @@ async function sendHelloTo12345() {
 ```
 
 Basically, all parameters of the function signature are merged with the options object when you use the raw API.
+
+## Choosing a Data Center Location
+
+> [Skip](./filter-queries) the rest of the page if you are just getting started.
+
+If you want to reduce the networking latency of your bot, it matters where you host it.
+
+The Bot API server behind `api.telegram.org` is hosted in Amsterdam in the Netherlands.
+Therefore, the best location to run your bot is Amsterdam.
+
+::: tip Hosting Comparison
+You may be interested in our [comparison on hosting providers](../hosting/comparison#comparison-tables).
+:::
+
+However, there might be an even better place to run your bot, although this takes significantly more effort.
+
+[Remember](#general-information) that the Bot API server does not actually contain your bot.
+It only relays requests, translates between HTTP and MTProto, and so on.
+The Bot API server may be in Amsterdam, but the Telegram servers are distributed across three locations: Amsterdam, Miami, and Singapore.
```

Comment:
```suggestion
The Bot API server may be in Amsterdam, but the Telegram servers are distributed across three different locations:

- Amsterdam, Netherlands
- Miami, Florida, United States
- Singapore
```

Feel free to ignore. Mainly suggesting that because one might not know where Miami is at a glance.

Reply:
I was unsure if we should do it, but now that I see it, I like it